components/bases-locales/validator/report/row-issues.js

```diff
@@ -3,12 +3,16 @@ import PropTypes from 'prop-types'
 
 import theme from '../../../../styles/theme'
 
-const RowErrors = ({errors, field}) => (
+const RowErrors = ({errors, warnings, field}) => (
   <div className='abnormalities'>
-    <h3>Anomalie{errors.length > 1 ? 's' : ''} :</h3>
+    <h3>Anomalie{(errors.length + warnings.length) > 1 ? 's' : ''} :</h3>
     <div className='error-list'>
+      {warnings.map(err => (
+        <div key={err} className={`issue warning ${field && (field.warnings.includes(err)) ? 'select' : ''}`}>
```

Comment:
Keys should be unique so that components maintain their identity across updates

Reply:
?

Reply:
`err` peut être identique sur plusieurs `key`